server/v2/go.mod

```diff
@@ -16,6 +17,8 @@ require (
 	cosmossdk.io/api v0.7.5
 	cosmossdk.io/core v0.12.1-0.20231114100755-569e3ff6a0d7
 	cosmossdk.io/log v1.3.1
+	cosmossdk.io/server/v2/appmanager v0.0.0-00010101000000-000000000000
+	github.com/cometbft/cometbft v0.38.7
```

Comment:
we should remove this dep,

Reply:
I haven't noticed it was introduced 👍🏾, we should remove indeed